### PR TITLE
[PWD-2458] Login Page

### PR DESCRIPTION
**Pull request by haikal1223**
### What type of PR is this? (_check all applicable_)

- [x] Feature
- [ ] Bug Fix
- [ ] Documentation Update
- [ ] Setup
- [ ] Code Refactor
- [ ] Performance Improvements
- [ ] Test
- [ ] Build
- [ ] CI
- [ ] Revert

### Screenshots (_optional_)

image

### CODE REVIEW

**Comment by shadiqfauzi** [comment]
sertakan screenshot ya

File: src/routes/index.tsx [hunk 1]
```diff
@@ -15,13 +16,16 @@ import { LearningMaterial } from '~/pages/LearningMaterial'
 import { Login } from '~/pages/Login'
 import { Register } from '~/pages/Register'
 import { Voucher } from '~/pages/Voucher'
-
-const isAuthenticatedExample = true
+import { IRootState } from '~/redux/rootReducer'
 
 const PublicRoutes = () => {
   // check auth state here
   // const { isAuthenticated } = useContext(AuthContext)
```
Comment (kuntiarso) [inline src/routes/index.tsx:23]:
```suggestion
```

File: src/routes/index.tsx [hunk 2]
```diff
@@ -32,6 +36,9 @@ const PublicRoutes = () => {
 const ProtectedRoutes = () => {
   // check auth state here
   // const { isAuthenticated } = useContext(AuthContext)
```
Comment (kuntiarso) [inline src/routes/index.tsx:38]:
```suggestion
```

File: src/views/login/section/FormSection.tsx [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+  const formik = useFormik({
+    initialValues: {
+      email: '',
+      password: '',
+      rememberMe: false
+    },
+    validationSchema,
+    onSubmit: () => {
+      dispatch(exampleSignIn())
+    }
```
Comment (kuntiarso) [inline src/views/login/section/FormSection.tsx:45]:
```suggestion
    onSubmit: () => dispatch(exampleSignIn())
```

File: src/routes/index.tsx (hunk 2 above)
Comment (haikal1223) [reply src/routes/index.tsx:38]: di take out ya? okee

**Review by kuntiarso — DISMISSED** [review]
sosis 👍

**Comment by haikal1223** [comment]
> sosis 👍

thanks boneless chick 😎

**Review by kuntiarso — DISMISSED** [review]
🚀🚀

**Review by kuntiarso — DISMISSED** [review]

File: src/main.tsx [hunk 4]
```diff
@@ -9,6 +11,8 @@ import './styles/index.css'
 
 root.render(
   <StrictMode>
-    <App />
+    <Provider store={store}>
```
Comment (shadiqfauzi) [inline src/main.tsx:14]: pindahin ke app aja providernya

Comment (shadiqfauzi) [reply src/main.tsx:14]: provider2 rata2 disitu

Comment (shadiqfauzi) [reply src/main.tsx:14]: provider2 rata2 disitu

Comment (haikal1223) [reply src/main.tsx:14]: okeehh

**Pull request merged**